app/controllers/unauthorized_controller.rb

```diff
@@ -32,8 +32,6 @@ def respond
   private
 
   def redirect_back_or(path)
-    redirect_to :back
-  rescue ActionController::RedirectBackError
-    redirect_to path
+    redirect_back fallback_location: path
```

Comment:
kinda makes this method obsolete, we can clean it up later to make the diff here smaller